### DISCUSSION TITLE
如何给stencil中节点增加点击事件

### DISCUSSION THREAD

**Discussion by LXY12121234** [antvis/X6#2449] [Q&A]
除拖拽功能以外，请问如何给stencil中节点绑定一个双击事件？求教。
<img width="173" alt="1659409877960" src="https://user-images.githubusercontent.com/52768004/182283748-2fc1448d-604e-473a-a962-eb386a478ab7.png">

**Comment by NewByVector** [comment]
暂时没有办法来实现。
